src/relay.rs

```diff
@@ -63,7 +61,7 @@ impl RelayNode {
         swarm.listen_on(relay_tcp_addr)?;
         swarm.listen_on(relay_mem_addr)?;
 
-        let task = spawn(swarm.for_each_concurrent(None, | _ | async move {}));
+        let task = spawn(swarm.for_each_concurrent(None, |_| async move {}));
         Ok(Self { port, task, id })
```

Comment:
So I was wrong when I said we could omit the relay's peer ID, right?

Reply:
not entirely, it seems we can omit it on the inside when listening on the relay (just call listen with `/p2p-circuit`). Furthermore, the [spec seems to support the omission (see "using any relay available")](https://github.com/libp2p/specs/blob/master/relay/circuit-v1.md) but when adding the addr or dialing for the first time (from the outside) it does/did seem to need the relay id when testing.